macOS/DuckDuckGo/TabBar/View/TabBarViewController.swift

```diff
@@ -1220,6 +1221,31 @@ extension TabBarViewController: TabBarViewItemDelegate {
         tabCollectionViewModel.tabViewModel(at: indexPath.item)?.tab.killWebContentProcess()
     }
 
+    func tabBarViewItemDidUpdateCrashInfoPopoverVisibility(_ tabBarViewItem: TabBarViewItem, sender: NSButton, shouldShow: Bool) {
+        guard shouldShow else {
+            crashPopoverViewController?.dismiss()
+            return
+        }
+
+        DispatchQueue.main.async {
+            let viewController = PopoverMessageViewController(
+                title: "This tab has crashed",
```

Comment:
Same question as the last one, are we going to add translations for these strings?